src/main/java/de/cas/mse/exercise/diamond/Diamond.java

```diff
@@ -4,28 +4,44 @@
 
 public class Diamond {
 
-	public String print(int n) {
-		if (n <= 0 || n % 2 == 0) {
+	public String createDiamond(int diamondSize) {
+		if (diamondSize <= 0 || diamondSize % 2 == 0) {
 			return null;
-		}
-		StringBuilder builder = new StringBuilder(new String(make(n, n)));
-		for (int i = n - 2; i > 0; i -= 2) {
-			char[] chars = make(n, i);
-			builder.insert(0, chars);
-			builder.append(chars);
-		}
-		return builder.toString();
+        }
+        
+        StringBuilder diamondBuilder = new StringBuilder(new String(createLine(diamondSize, diamondSize)));
+        
+		for (int i = diamondSize - 2; i > 0; i -= 2) {
```

Comment:
Please rename the "i" variable as well since it is ambiguous

Reply:
+1